app/concepts/reservations/use_cases/create_online.rb

```diff
@@ -0,0 +1,33 @@
+module Reservations
+  module UseCases
+    class CreateOnline  < Reservations::UseCases::CreateReservationWithTickets
+      attr_reader :repository
+
+      def initialize(repository: Reservations::Repository.new)
+        @repository = repository
+      end
+
+      def call(params:)
+        super
+        send_email
+        @reservaton
+        
+      end
+
+      def send_email
+        client = Clients::Repository.new.find_by(@reservation.client_id)
+        screening = Screenings::Repository.new.find_by(@reservation.screening_id)
+        movie = Movies::Repository.new.find_by(screening.movie_id)        
+
+        ReservationMailer.confirmation_email(
+          reservation: @reservation, 
+          email: client.email, 
+          date: screening.date,
+          movie: movie.title
```

Comment:
I'm not sure is correct but you have key `movie` but as a parameter sending `movie.title`

Reply:
So it would be better to call key `title` ?